src/data/queries/searchDoiQuery.ts

```diff
@@ -23,6 +23,10 @@ export function buildQuery(variables: QueryVar): string {
     buildOrgQuery(variables.rorId),
     variables.language ? `language:${variables.language}` : '',
     variables.registrationAgency ? `agency:${variables.registrationAgency}` : '',
+    variables.userId ? `creators.nameIdentifiers.nameIdentifier:*${variables.userId}*` : '',
```

Comment:
same here. looks like it has switched from `creators_and_contributors` to just `creators`